frontend/packages/dev-console/src/components/topology/TopologyView.tsx

```diff
@@ -0,0 +1,416 @@
+          <TopologyApplicationPanel
+            graphData={visualization.getGraph().getData()}
+            application={{
+              id: selectedEntity.getId(),
+              name: selectedEntity.getLabel(),
+              resources: selectedEntity.getData().groupResources,
+            }}
+          />
+        );
+      }
+      // TODO: Use Plugins
+      if (selectedEntity.getType() === TYPE_HELM_RELEASE) {
+        return <TopologyHelmReleasePanel helmRelease={selectedEntity} />;
+      }
+      if (selectedEntity.getType() === TYPE_HELM_WORKLOAD) {
+        return <TopologyHelmWorkloadPanel item={selectedEntity.getData() as TopologyDataObject} />;
+      }
+      if (selectedEntity.getType() === TYPE_OPERATOR_BACKED_SERVICE) {
+        return (
+          <TopologyOperatorBackedPanel
+            item={selectedEntity.getData() as TopologyDataObject<OperatorGroupData>}
+          />
+        );
+      }
+      if (selectedEntity.getType() === TYPE_VIRTUAL_MACHINE) {
+        return <TopologyVmPanel vmNode={selectedEntity} />;
+      }
+      return <TopologyResourcePanel item={selectedEntity.getData() as TopologyDataObject} />;
+    }
+
+    if (isEdge(selectedEntity)) {
+      if (selectedEntity.getType() === TYPE_EVENT_PUB_SUB_LINK) {
+        const itemResources = selectedEntity.getData();
+        return <KnativeResourceOverviewPage item={itemResources.resources} />;
+      }
+      if ([TYPE_REVISION_TRAFFIC, TYPE_EVENT_SOURCE_LINK].includes(selectedEntity.getType())) {
+        return <KnativeTopologyEdgePanel edge={selectedEntity as BaseEdge} />;
+      }
+      return <ConnectedTopologyEdgePanel edge={selectedEntity as BaseEdge} />;
+    }
+    return null;
+  };
+
+  const renderSideBar = () => {
+    if (!visualization || selectedIds.length === 0) {
+      return null;
+    }
+    const selectedEntity = visualization.getElementById(selectedIds[0]);
+    const details = selectedItemDetails();
+    if (!selectedEntity || !details) {
+      return null;
+    }
+
+    return (
+      <TopologySideBar show={!!selectedEntity && !!details} onClose={onSidebarClose}>
+        {selectedEntity && details}
+      </TopologySideBar>
+    );
+  };
+
+  const sideBar = renderSideBar();
+
+  const viewContent = showGraphView ? (
+    <Topology
+      visualization={visualization}
+      namespace={namespace}
+      application={application}
+      selectedIds={selectedIds}
+    />
+  ) : (
+    <TopologyListView
+      visualization={visualization}
+      namespace={namespace}
+      application={application}
+      selectedIds={selectedIds}
+      onSelect={onSelect}
+    />
+  );
+
+  const containerClasses = classNames('pf-topology-container', {
+    'pf-topology-container__with-sidebar': sideBar,
+    'pf-topology-container__with-sidebar--open': sideBar,
+  });
+
+  return (
+    <div className="odc-topology">
+      <Stack>
+        <StackItem isFilled={false}>
+          <TopologyFilterBar onSearchChange={onSearchChange} />
```

Comment:
Need to pass `visualization` props for graph view here to show the info icon next to the search field.

Reply:
We don't want the icon in the list view as the info presented is only appropriate for the graph view.

Reply:
Even in the Graph view icon is not visible.